src/Plugin/jsonapi/FieldEnhancer/CardLinkEnhancer.php

```diff
@@ -94,14 +99,16 @@ public function getCardFields($nid) {
     $card_fields['node_type'] = $node_type ? $node_type : '';
     // Add topic from the node.
     $topic_id = $node->hasField('field_topic') ? $node->get('field_topic')->getValue()[0]['target_id'] : '';
```

Comment:
Hi @MdNadimHossain , you could use the `$node->field_topic->isEmpty()` to check if the field empty or not.